styles.css

```diff
@@ -4,17 +4,19 @@
   padding: 0;
   box-sizing: border-box;
   font-size: 18px;
-  font-family: sans-serif;
+  font-family: 'Poppins', sans-serif;
   letter-spacing: 0.05rem;
 }
 
 :root {
   --color-base: #f5f5f5;
-  --color-highlight: #e7717d;
+  --color-highlight: #036b8e;
   --color-text: #1d3557;
   --color-dark: #7e685a;
-  --color-black: #000000;
-  --color-white: #ffffff;
+  --color-white: #fff;
+  --color-black: #0f0f1b;
+
+  --color-header: #bbc2cd;
```

Comment:
Added a new color for the header to accomadate the logo color which is black so they compliment each other. Our orginal highlight color didn't look as good